### ISSUE TITLE
Self-organizing through Open Collective Europe

### ISSUE THREAD

**Issue by fweilih** [rchain-community/rgov#283]
Whether it is to support the continued development or continue the operation of Mainnet, at the moment there is a pressing need to self-organize to raise donations.  This would require a certain minimum legal infrastructure including bank accounts and and reporting.  Open Collective (https://opencollective.com/) seems to offer such infrastructure to self-organizing groups.  

In particular Open Collective Europe (https://opencollective.com/europe), which is one of the 'fiscal hosts' which can support groups, does not require the group to have any pre-existing legal registration.  However, Open Collective Europe can only host EU-based projects and fund EU-based activity.  

The other fiscal host listed in the website is Open Source US, but that requires that the group have an existing registration as a not-for-profit association. 

In other words, whilst donations can come from anywhere, there is a need for European based community members to set up a Collective with Open Collective Europe if we wish to continue to support development of Rchain.

**Comment by jimscarver** [comment]
Effective governance of the rchain community is essential and opencollective.com seems a good way to go. All the EU members here, in https://t.me/rchain_coop  https://discord.gg/33x5eDk https://github.com/rchain-community/rgov/graphs/contributors and  https://github.com/rchain-community/rchain/graphs/contributors can be invited initially. Then the rest of us can join. While officially EU payments can be made to outside the EU and it may have members outside the EU.

I suggest interested members join the defunct Rchain Europe discord https://discord.gg/33x5eDk

@tgrospic can we set up a "governance" repo?

**Comment by peteratomic** [comment]
I'm in the EU and would join a group of other EU based members to do the application.

**Comment by fweilih** [comment]
I made contact with Open Source US and they clarified that a collective does not need to be a registered not-for-profit entity, and they will clarify the wording on their website going forward.

**Comment by fweilih** [comment]
Hi, this is the draft/work-in-progress budget that Jim and I put together today (most of credit to Jim, I was just the scribe).  Suggestions for improvement would be welcome. 

https://docs.google.com/spreadsheets/d/17PQ-uPuy5MUqeykeajmN9Gf2NdJgyFSaMciMCDNZxow/edit?usp=sharing

**Comment by peteratomic** [comment]
Looks good so far! Maybe $10-$20K for the new org research?

**Comment by iXiaowu2020** [comment]
The co-op members get so much money, why don't they contribute a penny?

**Comment by iXiaowu2020** [comment]
ross，darryl，rao，camlila，ting，where are they？

**Comment by iXiaowu2020** [comment]
You don't need the community, they're speculators, they don't care about the project, only the co-op members really care about the project

**Comment by iXiaowu2020** [comment]
Rchain is a compliance program. Money doesn't matter. The US government will bail it out

**Comment by jimscarver** [comment]
Former board members are necessarily quiet because of potential liabilities.

They are part of the community which we hope to Provide some organizational framework here now that the Co-Op is dissolved.

**Comment by fweilih** [comment]
> Looks good so far! Maybe $10-$20K for the new org research?

Thanks Peter, I will add that.

**Comment by jimscarver** [comment]
https://opencollective.com/rchain-community  Has been approved and is being sponsored by the open collective.

 I donated 1.8 ETH https://ethplorer.io/tx/0xc9439c91cdd09445c446a21d8c198335751406952b5043708dd0af5e63b0b16e

**Comment by tunkas** [comment]
Croatian community has prepared a donation of 9 ETH, but before sending it we would like to receive assurances that all expenses will be transparently shared with the community.
Who is holding the keys to the wallet(s), Jim?

**Comment by fweilih** [comment]
> Croatian community has prepared a donation of 9 ETH, but before sending it we would like to receive assurances that all expenses will be transparently shared with the community. Who is holding the keys to the wallet(s), Jim?

Our project is managed on the Open Collective platform ([Raise and spend money with full transparency. - Open Collective](https://opencollective.com/)), whilst the fiscal host is Open Source Collective ([Welcome! - Open Source Collective (oscollective.org)](https://docs.oscollective.org/)), a non-profit organization.  

That means that donations are held not by any of the administrators of the RChain Open Collective project, but rather by Open Source Collective.  You can read more about Open Source Collective here: [What is OSC? - Open Source Collective (oscollective.org)](https://docs.oscollective.org/about/what-is-osc).

Open Source Collective will deduct Dues and Expenses from all contributions, they are as follows [Open Source Collective Terms of Sponsorship Agreement (google.com)](https://docs.google.com/document/u/1/d/e/2PACX-1vQbiyK2Fe0jLdh4vb9BfHY4bJ1LCo4Qvy0jg9P29ZkiC8y_vKJ_1fNgIbV0p6UdvbcT8Ql1gVto8bf9/pub):

![image](https://user-images.githubusercontent.com/12480560/193959337-5080d6de-4bf1-4f72-a21b-4f1bb1d075af.png)
 
15% of the Dues to the Open Source Collective goes towards the Open Collective platform (see [Pricing - Open Collective Docs](https://docs.opencollective.com/help/about/pricing)).
 
![image](https://user-images.githubusercontent.com/12480560/193959082-126319a4-f503-4266-8826-4f6aa3a73450.png)

The Open Collective Platform is currently trialing crypto contributions, which is an available option since we are using Open Source Collective as our fiscal host.  At the moment, accepted cryptocurrencies are BTC, ETH, DAI, BCH, LTE and ZEC.  

Please be aware that crypto contributions are immediately converted into fiat.  See [Crypto Contributions - Open Collective Docs](https://docs.opencollective.com/help/financial-contributors/crypto).  That means that the RChain Collective administrators do not hold any private keys for any crypto accounts that receive contributions. 

The Open Source Collective supports a variety of other payment methods to make contributions, including credit card, PayPayl and bank transfer. See [General FAQ - Open Source Collective (oscollective.org)](https://docs.oscollective.org/faq/general).

![image](https://user-images.githubusercontent.com/12480560/193959114-350463cd-999b-4c28-a51f-9ffab0b33cb3.png)
 
Contributions are denominated in USD.

We will be using the functionality of the Open Collective platform to provide transparency regarding how funds are spent.  See [Transparent Budget - Open Collective Docs](https://docs.opencollective.com/help/collectives/budget).

In addition we are further looking into how to assess community sentiment when there are alternatives on how to spend funds to achieve our goals.

**Comment by fweilih** [comment]
Here is a follow-up question that was asked in Telegram:

"Who will be managing the budget?"

**Comment by fweilih** [comment]
> Here is a follow-up question that was asked in Telegram:
> 
> "Who will be managing the budget?"

In addition, we are looking into tools to help assess community sentiment when there are alternatives on how to spend funds to achieve our goals.   The principles we will follow when making decisions, including those pertaining to the budget, are described here: 

https://opencollective.com/rchain-community/conversations/governance-dpbvj0qz

You can see all of the donations collected, host fees deducted and expenses paid (none as of the date of this message), on the Open Collective Platform.  Click into the project below, and then choose ‘Budget’.

https://opencollective.com/rchain-community

https://opencollective.com/rchain-community/projects/continue-mainnet

**Comment by fweilih** [comment]
Here is another question from Telegram:  "What if US based OpenCollective decides they won't/can't support crypto project as RChain and thus cut the funding?"

The terms of the Open Source Collective Terms of Sponsorship Agreement, can be found here:

https://docs.google.com/document/u/1/d/e/2PACX-1vQbiyK2Fe0jLdh4vb9BfHY4bJ1LCo4Qvy0jg9P29ZkiC8y_vKJ_1fNgIbV0p6UdvbcT8Ql1gVto8bf9/pub

The answer to your question is found in Clause 6 of the terms, which reads:

"Either OSC or Collective may terminate this Agreement on 30 days' written notice to the other party. If there is a positive balance in the Project Fund or OSC holds Other Assets of the Project at the time of termination, the Project’s assets will be distributed as follows:

1.    OSC will reimburse any valid expenses of the Project upon receipt of a final request from Collective, submitted within 15 days of notice of termination, detailing and documenting the expense as specifically set forth in the Protocols.

2.    Within the 30 day notice period, Collective may identify a successor organization who is willing and able to carry on the Project, and (i) which is recognized as tax-exempt under Internal Revenue Code (the “Code”) section 501(c)(3) or 501(c)(6) or (ii) to which distribution of the Project Fund and Other Assets is otherwise consistent with applicable tax and charitable trust laws (a “Successor”). Upon determining that the Successor meets these qualifications and that the distribution is permitted by applicable law, OSC will transfer the balance of assets in OSC’s restricted Fund for the Project, together with any Other Assets held or liabilities incurred by OSC in connection with the Project, to the Successor. To initiate the transfer, the Successor payee must submit an expense as outlined in the Protocols. The transfer will be made after the end of the notice period or any extension thereof, subject to the approval of any third parties that may be required.

3.    As an alternative to a Successor, Collective may choose to transfer the balance of its Fund to another OSC Collective’s Fund, or to OSC’s general operating Fund, subject to applicable law.

4.    If no Successor or other qualified recipient is identified prior to the end of the notice period (including any mutually agreed extensions), OSC may, in its sole discretion, distribute or spend the Project Fund and dispose of Other Assets in any manner consistent with applicable tax and charitable trust laws, which may include granting any balance to another fiscal sponsor, re-allocating any balance to another Collective for substantially similar purposes, or using any balance to defray OSC’s costs of administering its fiscal sponsorship program."
